include/materials/CauchyStressFromNEML.h

```diff
@@ -23,6 +23,15 @@ class CauchyStressFromNEML : public ComputeLagrangianStressCauchy
 public:
   static InputParameters validParams();
   CauchyStressFromNEML(const InputParameters & parameters);
+  /// Reset some subset of the internal state
+  virtual void reset_state(const std::vector<unsigned int> & indices, unsigned int qp);
+  /**
+   * Get the offsets into the NEML state either by finding them
```

Comment:
The last half of the sentence that starts on line 29 got cut off -- just make sure this flows correctly.